src/app_modes/viewport.rs

```diff
@@ -254,6 +254,25 @@ impl UseViewport for Viewport {
         for line in Viewport::get_frame_lines(&base_link_pose, self.axis_length) {
             ctx.draw(&line);
         }
+
+        for pose_stamped in &self.listeners.pose_stamped {
```

Comment:
we probably can get rid of the repetitive lines by iterating over line listeners?

Reply:
Sort of. The order counts because stuff is drawn always on top (which can also be better enforced by layers). There is a specific reason why I wanted pose_array on top of path for example. If you prefer I'll collect every line in an array first, but that would remove the possibility of dividing by layers in the future. What do you think? (your way would be definitely nicer to read though ;) )